plugins/org.jboss.tools.cdi.reddeer/src/org/jboss/tools/cdi/reddeer/validators/ScopeValidationProviderCDI20.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+
+package org.jboss.tools.cdi.reddeer.validators;
+
+import java.util.Arrays;
+
+import org.eclipse.reddeer.eclipse.ui.views.markers.ProblemsView.ProblemType;
+import org.jboss.tools.cdi.reddeer.annotation.ValidationType;
+
+public class ScopeValidationProviderCDI20 extends AbstractValidationProvider {
+	
+	private static final String JSR = "JSR-365"; 
+
+	public ScopeValidationProviderCDI20() {
+		super();
+	}
+	
+	@Override
+	void init() {
```

Comment:
Same as the other comment.

Reply:
I have checked the messages and they are correct.